src/options.rs

```diff
@@ -193,7 +198,15 @@ impl Options {
         tmp.wait_until_available.map(|w| conn_params.wait_until_available(w));
 
         let unix_host = host.contains("/");
-        if admin || unix_host {
+        if let Some(name) = tmp.instance {
+            let path = server::get_instance(&name)
+                .and_then(|inst| inst.get_socket(admin))
+                .unwrap_or_else(|e| {
+                    eprintln!("Failed to find instance {:?}: {:#}", name, e);
```

Comment:
s/Failed/Could not/ and I'd also add a hint that instances are expected to be created using `edgedb server init`.